components/ciphertool/src/main/java/org/wso2/ciphertool/CipherTool.java

```diff
@@ -89,7 +89,7 @@ private static void initialize(String[] args) {
                 } else if ((Constants.CHANGE).equals(propertyName)) {
                     System.setProperty(property, Constants.TRUE);
                 } else if ((Constants.CIPHER_TRANSFORMATION_SYSTEM_PROPERTY).equals(propertyName)) {
-                    if (!Utils.isBlank(value)) {
+                    if (!value.isEmpty()) {

```

Comment:
java.lang.String.isEmpty() return false for null. So we should use org.apache.commons.lang3.StringUtils class's isEmpty() or isBlank()

Reply:
Did not add that method because that dependency was not in the repo earlier.
Added the dependency from the latest commit.

Reply:
@RameshaKaru you can explicitly do a null check an isEmpty in this case instead of adding a new dependency (commons-lang3) only to use the StringUtils.isBlank.

Reply:
As discussed with @rmsamitha, we believe adding commons-lang3 dependency will be useful for future developments and additions too

Reply:
Noted - but IMO it would be better add it when the need arises since adding new dependencies will bring in overheads such as upgrading it if there is a security vulnerability, increase jar size, etc...

Reply:
@Niranjan-K  Agreed on some possible overheads and thanks for the feedback. My perception is that, it is now that the need for this dependency has risen. If we do not add a dependency when needed as now, (for even minor requirements), then we may never add it in the future too, but will keep adding those code manually forever when needed.
And even I see multiple places in this same repo, where StringUtils.isEmpty can be used instead of String.isEmpty which will throw errors at null cases.

Reply:
BTW most of the products using cipher-tool use this additional dependency org.apache.commons.lang3 already, so security vulnerabilities will have to be anyway addressed. And even for such these widely used dependencies, we can disregard on rarely possible security vulnerabilities AFAIU.
And we do not need to pack the external dependency jars with cipher tool jar, isn't it? (so the jar size won't matter).
So adding this particular dependency would be better considering other general overheads, isn't it ?

Reply:
By adding this library, we need to add new dependencies for all the product repository as well. IMO I think better to implement it in the Utill class and reuse it over this repository. It is not worth to get whole library to do only this task.

Reply:
@madushadhanushka We use this dependency and pack it in most products already as mentioned previously too.

Reply:
@rmsamitha understood that it is being used in other products, but in the case of cipher tool, AFAIK it is a standalone jar, therefore the dependencies will be packed into it. 
Agreed that rarely security vulnerabilities for these components, but IMO it should not be disregarded.
My suggestion was that we can directly do the check instead of adding a dependency to do the same (in this case).

Reply:
@Niranjan-K They are not packed in cipher-tool jar, but has references in pom, so will be packed in the end product.